### ISSUE TITLE
Threema login information

### ISSUE THREAD

**Issue by mfulz** [hbruintjes/ceema#6]
I've compiled the pidgin plugin and could basically add a Threema account.
But how / what do I need to enter for Username / Password?

**Comment by hbruintjes** [comment]
Usually, it is your Threema ID for username, the password for a backup string as password, and then in the account settings specify the backup string. You can fetch the backup string from the app.
Alternatively you can directly use a hex-encoded version of your private key as the password and leave the backup string blank, but you would have to extract it first.

**Comment by mfulz** [comment]
Ok found it, but still have the following issues:

- Default server g-xx.0.threema.ch does not exists (I think xx can be replaced by numbers)
- When using different numbers (xx = 01, 10, 22, 05, etc.) I'm receiving error 16:
(12:13:45) threepl: TRACE - Packet ready, state 1
(12:13:45) threepl: TRACE - Sending session PK: 
(12:13:45) threepl: TRACE - Sending nonce: 
(12:13:45) threepl: TRACE - onReadyWrite 48
(12:13:45) threepl: TRACE - Written 48 bytes
(12:13:45) threepl: TRACE - Packet handled, state 2, remaining data 0
(12:13:45) threepl: TRACE - onReadyWrite 0
(12:13:45) threepl: TRACE - onReadyRead socket
(12:13:45) threepl: TRACE - Read 80 bytes, expecting 80
(12:13:45) threepl: TRACE - Packet ready, state 2
(12:13:45) threepl: TRACE - Got server nonce: 
(12:13:45) connection: Connection error on  (reason: 16 description: Unable to decrypt HELLO packet)

I've removed thenonces and session PK, because I'm not sure how critical this data is.

**Comment by hbruintjes** [comment]
Seems Threema has shut the 3rd-party application gateway. You can replace the 'xx' by one of the alternatives, but make sure to uncheck "Use application gateway" in the settings, as those gateways use a slightly different protocol.

**Comment by mfulz** [comment]
Yep it's not working I think, with your suggestions I receive account disabled because you signed in from another location.
So I would have to disable it on my phone I guess.

**Comment by mfulz** [comment]
Ok just have to close threema on my phone than it connects.
But it is not showing any contacts -> Is this a missing feature or some issue on my side?

**Comment by hbruintjes** [comment]
In PR #4 some backup import feature was added, but I have never tried that. Automatic import is not supported (if the server even supports that, previously that did not exist). You will have to add them manually as buddies.
